src/Ryujinx.Ava/UI/ViewModels/MainWindowViewModel.cs

```diff
@@ -1184,6 +1196,15 @@ private void Update_StatusBar(object sender, StatusUpdatedEventArgs args)
                         VsyncColor = new SolidColorBrush((Color)color);
                     }
 
+                    if (args.ShaderCount > 0)
+                    {
+                        ShaderCountText = LocaleManager.Instance[LocaleKeys.CompilingShaders] + $": {args.ShaderCount}";
+                    }
+                    else
+                    {
+                        ShaderCountText = "";
+                    }
```

Comment:
```suggestion
                    ShaderCountText = args.ShaderCount > 0 ? LocaleManager.Instance[LocaleKeys.CompilingShaders] + $": {args.ShaderCount}" : "";
```

But maybe we should show another text instead of nothing if no shaders are compiling.

Reply:
Adjusted. 

I think it's better not to clog UI space if nothing is going to be shown. The point of this is to let people know there are/were active compiles if they notice a stutter. Once that period is over I would expect the message to go away.